rpcd/playbooks/roles/elasticsearch/tasks/upgrade_pre_flight.yml

```diff
@@ -0,0 +1,52 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: "Get ElasticSearch version"
+  command: "dpkg-query --show --showformat='{$Version}' elasticsearch"
+  register: elasticsearch_installed_version
+
+- name: "Check for ElasticSearch version > 2.x"
+  set_fact:
+    proceed_with_upgrade: false
+    latest_version: true
+  register: proceed_with_upgrade
+  when: "{{ elasticsearch_installed_version.stdout | version_compare({{ elasticsearch_version }}, '>=', strict=True) }}"
```

Comment:
Does the `{{ elasticsearch_version }}` work inside the `{{ }}`? Is it necessary?